server/routes/checkAnswers.js

```diff
@@ -118,61 +112,60 @@ const typeOfHandcuffsUsed = handcuffsType => {
-    Location: description,
-    'Use of force planned': newIncident.forceType,
-    'Staff involved': toTitleCase(convertArrayOfObjectsToStringUsingSpecifiedKey('name', involvedStaff)),
-    Witnesses: toTitleCase(convertArrayOfObjectsToStringUsingSpecifiedKey('name', newIncident.witnesses)),
+    offenderName: offenderDetail.displayName,
+    offenderNumber: offenderDetail.offenderNo,
+    location: description,
+    forceType: newIncident.forceType,
+    staffInvolved: convertArrayOfObjectsToStringUsingSpecifiedKey('name', involvedStaff),
+    witnesses: convertArrayOfObjectsToStringUsingSpecifiedKey('name', newIncident.witnesses),
   }
 }
 
 const createDetailsObj = (details = {}) => {
   return {
-    'Positive communication used': details.positiveCommunication,
-    'Personal Protection Techniques': details.personalProtectionTechniques,
-    'Baton drawn': whenPresent(details.batonDrawn, value =>
-      value === 'Yes' ? wasWeaponUsed(details.batonUsed) : 'No'
-    ),
-    'PAVA drawn': whenPresent(details.pavaDrawn, value => (value === 'Yes' ? wasWeaponUsed(details.pavaUsed) : 'No')),
-    'Guiding hold used': whenPresent(details.guidingHold, value =>
+    positiveCommunicationUsed: details.positiveCommunication,
+    personalProtectionTechniques: details.personalProtectionTechniques,
+    batonDrawn: whenPresent(details.batonDrawn, value => (value === 'Yes' ? wasWeaponUsed(details.batonUsed) : 'No')),
+    pavaDrawn: whenPresent(details.pavaDrawn, value => (value === 'Yes' ? wasWeaponUsed(details.pavaUsed) : 'No')),
+    guidingHoldUsed: whenPresent(details.guidingHold, value =>
       value === 'Yes' ? `Yes ${howManyOfficersInvolved(details.guidingHoldOfficersInvolved)}` : 'No'
     ),
-    'Control and restraint used': whenPresent(details.restraint, value =>
-      value === 'Yes' ? `Yes - ${getRestraintPositions(details.restraintPositions)}` : 'No'
+    controlAndRestraintUsed: whenPresent(details.restraint, value =>
+      details.restraintPositions && value === 'Yes' ? getRestraintPositions(details.restraintPositions) : undefined
     ),
-    'Handcuffs used': whenPresent(details.handcuffsApplied, value =>
+    handcuffsUsed: whenPresent(details.handcuffsApplied, value =>
       value === 'Yes' ? typeOfHandcuffsUsed(details.handcuffsType) : 'No'
     ),
   }
 }
 
 const createRelocationObj = (relocationAndInjuries = {}) => {
   return {
-    'Prisoner relocated to': relocationAndInjuries.prisonerRelocation,
-    'Relocation compliancy': relocationAndInjuries.relocationCompliancy,
-    'HealthCare staff present': whenPresent(relocationAndInjuries.healthcareInvolved, value =>
-      value === 'Yes' ? `${toTitleCase(relocationAndInjuries.healthcarePractionerName)}` : 'No'
+    prisonerRelocation: relocationAndInjuries.prisonerRelocation,
+    prisonerCompliancy: relocationAndInjuries.relocationCompliancy,
+    halthcareStaffPresent: whenPresent(relocationAndInjuries.healthcareInvolved, value =>
```

Comment:
tpyo

Reply:
should have caused the cyprus test to fail though :thinking:

Reply:
Change made and pushed up